docs/product/explore/feature-flags/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Feature Flags"
+sidebar_order: 100
+description: "Learn how to set up and interact with Sentry's feature flag evaluation tracking and feature flag change tracking."
+---
+
+<Alert level="info" title="Currently in Beta">
+
+The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in beta.
+
+</Alert>
+
+Link your external feature flag integrations with Sentry to provide feature flag insights right inside the Sentry UI. Linking one or more integrations will allow you to view recent flag evaluations in one place and identify potential suspect flags related to errors. In addition, Sentry will provide insights on feature flag updates relative to error event timelines.
+
+## Touchpoints
+
+Feature flag insights can be found by navigating to the Issue Details page for any error event. There are two places where you can interact with flag data:
+* Flag updates (changes, deletions, and additions) will appear as a series on the event and user volume chart.
+* Flag evaluations will appear in the Feature Flag section in Issue Details as a table, with "suspect" flag predictions highlighted in yellow.
+
+Learn more about how to interact with feature flag insights within the Sentry UI by reading the [Issue Details docs](/product/issues/issue-details/#feature-flags/).
+
+## Setup
+
+In order to take advantage of the full feature flag experience available, there are two steps necessary to set up **feature flag evaluation tracking** and **feature flag change tracking**. The first step — setting up the language-specific SDK — is necessary for evaluation tracking, and the second step — setting up your integration-specific webhook — is necessary for change tracking.
+
+## Set Up Your Language-Specific SDK
+
+In order to gain access to **feature flag evaluation tracking**, you need to set up your SDK to include Sentry's feature flag integration.
+
+Learn more about the languages we currently support and how to set up the SDK:
+* [JavaScript](/platforms/javascript/feature-flags/)
+* [Python](/platforms/python/feature-flags/)
+
+## Set Up Your Integration-Specific Webhook
+
+In order to gain access to **feature flag change tracking**, you need to set up a webhook with your specific integration, so that it may communicate feature flag changes with Sentry.
```

Comment:
```suggestion
In order to gain access to **feature flag change tracking**, you need to set up a webhook with your specific integration, so that it can communicate feature flag changes with Sentry.
```